community.html

```diff
@@ -95,6 +95,21 @@ <h2>Twitter</h2>
             <a href="https://twitter.com/search?q=%23prestodb">#prestodb</a>
             hashtag on Twitter to keep up with the latest Presto news.
         </p>
+
+        <h2>Slack</h2>
+
+        <p>
+            The Presto developer community is coordinating activities via
+            <a href="https://prestodb.slack.com">prestodb.slack.com</a>
```

Comment:
We point to the self-signup link, instead:

```
https://join.slack.com/t/prestodb/shared_invite/enQtNDczNzEzMjYxMTc0LTc5MTVmZjE1N2E4YjJmZDBlNDBkOGUwMjM5YWIwM2E1ZjI3M2FiYzk2MTRmZjFjYjQ3ZmIwNzAzZTU5MWM4ZGU
```

Reply:
We might want to get rid of the IRC channel reference above, too, since no one is really using it.

Reply:
Totally agree.  I don't even have an IRC client on my machine anymore.

Reply:
link href changed, IRC removed